app/javascript/chat/components/ChannelButton.jsx

```diff
@@ -6,22 +6,24 @@ import { ChannelImage } from './ChannelImage';
 import { Button } from '@crayons';
 
 /**
- * Render the channel button
+ * Render a button to switch focus to a channel
+ *
  * @param {object} props
- * 
```

Comment:
(suggestion / not blocking) - it would be helpful to add a breakdown of the prop types here while we're fleshing out the JSDoc for this component

Reply:
I considered refactoring the component to take specific parameters rather than one object. Perhaps I should do that in a follow-up?

Reply:
I love the idea of destructuring the object in a follow up